docs/Developers/Newcomers_Guide.rst

```diff
@@ -355,6 +355,9 @@ to be merged, create a commit and push it to your fork, using:
 
 where ``myfork`` is the name of your fork that you added at the previous step.
 
+After making the changes, you can test the code on your forked repository by
+adding the required CI tools. See `here <http://api.coala.io/en/latest/Developers/Adding_CI.html>`_.
```

Comment:
dont use "here".  Describe the link using normal words that actually describe the link target.

Also, put `See <long link>` on the next line, to keep lines under 80cpl wherever possible.